pipeline/run.sh

```diff
@@ -183,15 +178,15 @@ for mzML in ${outdir}/1-data/*.mzML ; do
     # 1-generateBreaksFwhm.HPC.R
     echo "#!/bin/sh
 
-    /hpc/local/CentOS7/common/lang/R/3.2.2/bin/Rscript ${scripts}/1-generateBreaksFwhm.HPC.R \$mzML ${outdir} ${trim} ${resol} ${nrepl} ${scripts}
+    singularity exec -B /hpc/dbg_mz:/hpc/dbg_mz dims.img Rscript ${scripts}/1-generateBreaksFwhm.HPC.R \$mzML ${outdir} ${trim} ${resol} ${nrepl} ${scripts}
```

Comment:
1. Are additional environmental parameters useful? such as: 

- SINGULARITY_CACHEDIR
- SINGULARITY_TMPDIR
- SINGULARITY_PULLFOLDER 
https://singularity-userdoc.readthedocs.io/en/latest/build_environment.html?highlight=cache#cache

2. Is tmpspace used by the R-script? In that case, you could bind that dir as well.  `-B $TMPDIR:$TMPDIR `